avaframe/com3Hybrid/com3Hybrid.py

```diff
@@ -78,14 +94,16 @@ def maincom3Hybrid(cfgMain, cfgHybrid):
         shpConv.writeLine2SHPfile(avaProfileMassExtOri, name, pathAB)
 
         # Run Alpha Beta
-        hybridModelABCfg = pathlib.Path('com3Hybrid', 'hybridModel_com2ABCfg.ini')
-        cfgAB = cfgUtils.getModuleConfig(com2AB, fileOverride=hybridModelABCfg)
-        cfgAB['ABSETUP']['path2Line'] = str(pathAB) + '.shp'
+        # first create configuration object for com2AB
+        com2ABCfg = cfgUtils.getModuleConfig(com2AB, fileOverride='', modInfo=False, toPrint=False,
+            onlyDefault=cfgHybrid['com1DFA_override']['defaultConfig'])
+        com2ABCfg, cfgHybrid = cfgHandling.applyCfgOverride(com2ABCfg, cfgHybrid, com2AB, addModValues=True)
```

Comment:
What about reading cfg and applying override at the top for everyone? I know it is not how I did it but it does not make sense to read them in the loop

Reply:
done